### PR TITLE
Fixes for CombiningBuilder

### PR DESCRIPTION
**Pull request by kevmoo**
Will land seperately

### CODE REVIEW

File: source_gen/lib/src/builder.dart [hunk 1]
```diff
@@ -106,13 +106,22 @@ class _Builder extends Builder {
                 'Consider adding the following to your source file:\n\n'
                 'library $suggest;');
       }
-      final part = computePartUrl(buildStep.inputId, outputId);
+      contentBuffer.writeln();
+
+      String part;
+      if (_outputPartOf) {
```
Comment (grouma) [inline source_gen/lib/src/builder.dart]: Discussed offline. These variable names are confusing and not indicative in what we are using them for.

Comment (kevmoo) [reply source_gen/lib/src/builder.dart]: right...but is this PR okay otherwise?

Comment (grouma) [reply source_gen/lib/src/builder.dart]: I was hoping you'd fix it in this one.

**Comment by kevmoo** [comment]
> I was hoping you'd fix it in this one.

I'll do a follow-up.

**Pull request merged**